tests/x11/firefox_audio.pm

```diff
@@ -18,16 +18,18 @@ use testapi;
 
 sub run {
     my ($self) = @_;
-    $self->start_firefox();
```

Comment:
Argument which you've mentioned is valid not for this scenario, but all scenarios in general, so I would rather change start_firefox method as it does some extra logic, like cleaning and redirecting logs to firefox.log file. But that's optional.